konan_sdk/sdk.py

```diff
@@ -19,88 +18,30 @@ def __init__(self, auth_url="https://auth.konan.ai", api_url="https://api.konan.
         self.password = ''
 
         self.user: Optional[KonanUser] = None
```

Comment:
`self.user` is redundant. You can simply use `self.auth.user` instead whenever `self.user` is referenced.
```suggestion
```